src/components/BrowserSync.js

```diff
@@ -43,7 +43,7 @@ class BrowserSync {
         let defaultConfig = {
             host: 'localhost',
             port: 3000,
-            proxy: 'app.test',
+            proxy: process.cwd().replace(/\\/g, '/').split('/').pop() + '.test',
```

Comment:
This would be a bit more concise:

```suggestion
            proxy: `${path.basename(process.cwd())}.test`,
```

Reply:
We'd want to require path at the top as well.

Reply:
Thanks